### DISCUSSION TITLE
Theme page looks weird with two headings

### DISCUSSION THREAD

**Discussion by bolatovumar** [btcpayserver/btcpayserver#3466] [General]
We have two headings on the theme page which are almost the same size and it looks like there should be some content between them but there is nothing. I wonder if one of the headings should just be removed? cc @dstrukt 

![theme](https://user-images.githubusercontent.com/1934678/154409608-651d8265-c851-4744-93a5-14d6e7f1533a.PNG)

**Comment by dstrukt** [comment]
Agreed, nice catch, removed:
https://github.com/btcpayserver/btcpayserver/pull/3470
